src/common/arttango.cpp

```diff
@@ -103,34 +106,41 @@ class wxTangoArtProvider : public wxArtProvider
 // implementation
 // ============================================================================
 
-wxBitmap
-wxTangoArtProvider::CreateBitmap(const wxArtID& id,
+wxBitmap wxTangoArtProvider::CreateBitmap(const wxArtID& id,
+    const wxArtClient& client,
+    const wxSize& size)
+{
+    wxBitmapBundle bb = CreateBitmapBundle(id, client, size);
+    if (bb.IsOk())
+        return bb.GetBitmap(size);
+    else
+        return wxNullBitmap;
+}
```

Comment:
This should be really done in `wxArtProvider::GetBitmap()` itself, this is not at all specific to Tango provider.

Reply:
I agree but by simply putting it in the current `wxArtProvider` implementation would that create a potential recursion? as the default `CreateBitmapBundle()` implementation also makes a call to `CreateBitmap()`.

Reply:
You're right, I think I made a mistake with implementing `CreateBitmapBundle()` like this. I've tried to fix it in #22078 now, which should make overriding `CreateBitmap()` here unnecessary, so if you don't disagree, I'll merge the other PR first and then merge this one without this part.